### PR TITLE
test: prepare for merge to azure-pipelines-tasks repo

### PR DESCRIPTION
**Pull request by JeyJeyGao**
Fix:
- update localization task.loc.json
- reset task version

Test:
- added unit test for install, sign, verify commands (required for azure-pipelines-tasks repo)

### CODE REVIEW

File: NotationV0/Strings/resources.resjson/en-US/resources.resjson [hunk 1]
```diff
@@ -18,17 +17,16 @@
...
+  "loc.input.help.artifactRefs": "Container artifact references for signing. If it was not specified, use the artifact reference from previous Docker push task. Example: <registry name>/<repository name>@<digest> If multiple, separate by comma.",
   "loc.input.label.signatureFormat": "Signature Format",
   "loc.input.help.signatureFormat": "Signature envelope format, options: \"jws\", \"cose\" (default \"jws\")",
   "loc.input.label.allowReferrersAPI": "[Experimental] Allow Referrers API",
   "loc.input.help.allowReferrersAPI": "Use the Referrers API to sign signatures, if not supported (returns 404), fallback to the Referrers tag schema.",
   "loc.input.label.plugin": "Plugin",
-  "loc.input.label.azureKeyVault": "Azure Key Vault Plugin",
   "loc.input.label.akvPluginVersion": "Plugin Version",
-  "loc.input.help.akvPluginVersion": "The version of the Azure Key Vault plugin to be installed. Please visit the release page for the available versions.",
+  "loc.input.help.akvPluginVersion": "The version of the Azure Key Vualt plugin to be install. please visit the [release page](https://github.com/Azure/notation-azure-kv/releases) for the available versions.",
```
Comment (shizhMSFT) [inline NotationV0/Strings/resources.resjson/en-US/resources.resjson]: `Vualt` --> `Vault`?

Comment (shizhMSFT) [reply NotationV0/Strings/resources.resjson/en-US/resources.resjson]: `install` --> `installed`?

Comment (JeyJeyGao) [reply NotationV0/Strings/resources.resjson/en-US/resources.resjson]: Updated.

**Review by shizhMSFT — APPROVED** [review]
LGTM

**Pull request merged**